### PR TITLE
homework for meal sharing react 2 week1

### PR DESCRIPTION
**Pull request by aladdin91**
this is my h/w for meal sharing app react week2

### CODE REVIEW

**Comment by shpomp** [comment]
Which homework is this? 

React2 week2 asks for:
image


Which is not in this PR.

The branch and PR names are very confusing  😵

**Comment by aladdin91** [comment]
I’m so sorry for the confusing names, this is the meal sharing react 2 week1, i will edit the name of the PR

File: src/client/components/Meal.jsx [hunk 1]
```diff
@@ -0,0 +1,13 @@
+import React from "react";
+import "./meal.css";
+function Meal({ meal }) {
+  return (
```
Comment (shpomp) [inline src/client/components/Meal.jsx:5]: Try destructuring the meals prop object for tis properties 💡

File: src/client/components/MealList.jsx [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+    } catch (err) {
+      console.error(err);
+    }
+  }, []);
+
+  return (
+    <div>
+      <h2>List of meals</h2>
+      <div className="continer">
+        {meals.map((meal) => (
```
Comment (shpomp) [inline src/client/components/MealList.jsx]: Consider the case where meals are not fetched yet, so that this does not throw an error 💡

**Review by shpomp — APPROVED** [review]
Very nice! 👍🏽 
A few ideas for improvement 🙏🏽

**Comment by aladdin91** [comment]
thanks a lot @shpomp for the feedback 🙏

**Pull request merged**